### DISCUSSION TITLE
Ran in to this issue while building the project

### DISCUSSION THREAD

**Discussion by naveenrpendli** [ConsenSys/quorum#1351] [Q&A]
Hello Team,

I have run into this issue and stuck at his point can someone provide me some inputs on resolving the issue. Any help will be greatly appreciated. Thanks in advance.

--Naveen

![image](https://user-images.githubusercontent.com/25126027/156911050-585b6a27-0aeb-426d-8503-4d064dab19da.png)
.

**Comment by antonydenyer** [comment]
What version of go do you have installed? What is the result of `make` from the cli?
